src/main/resources/data.sql

```diff
@@ -140,13 +140,13 @@ insert into student (first_name,last_name,group_id,english_level_id) values ('Th
 insert into student (first_name,last_name,group_id,english_level_id) values ('Hans','Landa',3,2);
 insert into student (first_name,last_name,group_id,english_level_id) values ('King','Schultz',3,3);
 insert into student (first_name,last_name,group_id,english_level_id) values ('Ace','Ventura',3,4);
-insert into student (first_name,last_name,group_id,english_level_id) values ('Sarah','Connor',4,2);
-insert into student (first_name,last_name,group_id,english_level_id) values ('Vito','Corleone',4,3);
-insert into student (first_name,last_name,group_id,english_level_id) values ('Luke','Skywalker',4,2);
-insert into student (first_name,last_name,group_id,english_level_id) values ('Rick','Deckard',4,4);
-insert into student (first_name,last_name,group_id,english_level_id) values ('Anton','Chigurh',4,1);
-insert into student (first_name,last_name,group_id,english_level_id) values ('Rocky','Balboa',4,4);
-insert into student (first_name,last_name,group_id,english_level_id) values ('Jules','Winnfield',4,4);
+-- insert into student (first_name,last_name,group_id,english_level_id) values ('Sarah','Connor',4,2);
+-- insert into student (first_name,last_name,group_id,english_level_id) values ('Vito','Corleone',4,3);
+-- insert into student (first_name,last_name,group_id,english_level_id) values ('Luke','Skywalker',4,2);
+-- insert into student (first_name,last_name,group_id,english_level_id) values ('Rick','Deckard',4,4);
+-- insert into student (first_name,last_name,group_id,english_level_id) values ('Anton','Chigurh',4,1);
+-- insert into student (first_name,last_name,group_id,english_level_id) values ('Rocky','Balboa',4,4);
+-- insert into student (first_name,last_name,group_id,english_level_id) values ('Jules','Winnfield',4,4);
```

Comment:
@Crutovel ++. Moreover, if we have students in the 'planned' group, we can't actually delete this group.